### PR TITLE
Support //go:build build flag in formatter CI

### PR DESCRIPTION
**Pull request by kevindiu**
Signed-off-by: kevindiu <kevindiujp@gmail.com>

### Description:

This PR updates formatter CI to support `//go:build` flag in formatter.

### Environment:

- Go Version: 1.17
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [x] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by hlts2 — APPROVED** [review]

File: internal/net/control/control_darwin.go [hunk 1]
```diff
@@ -1,4 +1,5 @@
 //go:build darwin && !linux && !windows && !wasm && !js
+
```
Comment (rinx) [inline internal/net/control/control_darwin.go:2]: I'm not sure but is it okay to insert a blank line here?

Comment (kevindiu) [reply internal/net/control/control_darwin.go:2]: I think it is better not to insert blank line here... I will update the source code

File: hack/license/gen/main.go [hunk 2]
```diff
@@ -178,101 +180,117 @@ func readAndRewrite(path string) error {
...
+		line = in.Text()
+		if !isBuildFlag(ext, line) {
+			if strings.HasPrefix(line, d.Escape) {
+				continue
+			}
+			break
+		}
+
+		if _, err := out.WriteString(line); err != nil {
+			log.Fatal(err)
```
Comment (hlts2) [inline hack/license/gen/main.go:243]: `log.Fatal` is called a lot of places. Should we return an error? 🤔

**Comment by kevindiu** [comment]
Seems I will stop working on this PR now. I will reopen it when I start working on it again

**Pull request closed without merging**